File: backend/src/main/java/dto/cmd/client/TurnEndedCmd.java
package dto.cmd.client;

import dto.DTOCode;
import dto.DTOs;
import io.netty.buffer.ByteBuf;


@DTOCode(DTOs.TURN_ENDED)
public class TurnEndedCmd extends Command {

    @Override
    public void execute () {

    }


    @Override
    protected void writeMembers (ByteBuf byteBuf) {

    }


    @Override
    protected void readMembers (ByteBuf buffer) {

    }
}

File: frontend/Assets/Scripts/DataTransfer/Client/TurnEndedCmd.cs
using System.IO;
using Attributes;
using UnityEngine;


namespace DataTransfer.Client {

    [DTO(DTOCode.TurnEnded)]
    public class TurnEndedCmd : ClientCommand {

        public bool Alive { get; private set; }


        public TurnEndedCmd (bool alive) {
            Alive = alive;
        }


        public override void WriteMembers (BinaryWriter writer) {
            writer.Write(Alive);
        }

    }

}
